src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -6,9 +6,11 @@ import { usePartygoers } from "hooks/users";
 import { currentVenueSelector, parentVenueSelector } from "utils/selectors";
 
 import "./VenuePartygoers.scss";
+import { usePartygoersThreshold } from "hooks/useCampPartygoers";
```

Comment:
Would be nice to have this in `hooks/users`

Reply:
This PR just hasn't been updated since your refactor hit `staging`, this is the old location of that hook, just needs to update it in `hooks/users` when it's updated with the latest in `staging`.